scripts/data_collector/utils.py

```diff
@@ -564,3 +562,47 @@ def generate_minutes_calendar_from_daily(
 
 if __name__ == "__main__":
```

Comment:
It will be better if `ifmain` is at the bottom of the script.

Reply:
Yes, much better!
It was a lack of attention on my part